etc/config-template.toml

```diff
@@ -1057,6 +1057,15 @@
 ## in order to reduce impact on the cluster's performance during back up.
 # enable-auto-tune = false 
 
+[backup-stream] 
+## Number of threads to perform backup stream tasks.
+## The default value is set to min(CPU_NUM * 0.5, 8).
+# num-threads = 24
```

Comment:
```suggestion
# num-threads = 8
```

This value should keep consistence to the default config🤔

But the default is variant by the runtime. So what number should there be?